README.md

```diff
@@ -90,6 +90,36 @@ Common options are described in some detail in solr/README.txt.  For an
 exhaustive treatment of options, run `bin/solr start -h` from the `solr/`
 directory.
 
+### Running Solr in Docker
+
+You can run Solr in Docker via the [official image](https://hub.docker.com/_/solr).
+
+To simply run Solr in a container and expose the Solr port, run:
```

Comment:
s/simply//

I learned this trick from a tech writer friend a while ago, that you never want to say simply, or just, or obviously, in the documentation. Because while it may be simple to you as the writer, it's probably not simple to the reader and that's why they are reading the docs in the first place. We don't need to insult our readers. :)